Project-II-Calculator/calculator/src/components/ButtonComponents/NumberButton.js

```diff
@@ -1,2 +1,38 @@
 import React from 'react';
 import './Button.css';
+function Button (){
+    return(
+        <div>
+        <div className="row">
+            <p className="wide">clear</p>
```

Comment:
For the formatting aspect, using 'p-tags' is okay. When considering functionality, you could wrap all of these in 'buttons' ... even if you leave all the attributes (onClick, etc) blank, you can format the buttons and set up for adding functionality in the future.